com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/PyFunctionSource/PyFunctionSource_cpp.cgt

```diff
@@ -11,6 +11,10 @@
 
 <%SPL::CodeGen::implementationPrologue($model);%>
 
+<%
+  # hard code tuple stype is being used since $pystyle needed in pywrapfunction.cgt  
+  my $pystyle = 'pickle';
```

Comment:
This is a little confusing because pystyle refers to the style of an input port and this is a source operator with no input port.

What changed to make this required?